docs/projects/amt/reporting-standard/latest.md

```diff
@@ -128,6 +128,16 @@ There can be multiple instruments. For each instrument the following fields are
         2. `state` (REQUIRED, string).
         3. `value` (REQUIRED, string). Description on how the measure is implemented.
         4. `version` (OPTIONAL, string). The version of the Algoritmekader.
+        5. `responsible_persons` (OPTIONAL, list). The persons responsible for execution of this measure.
```

Comment:
A little bit nitty gritty, but I would change the order to accountable, responsible, reviewer. Makes it a bit more intuitive to read.

Reply:
Agree, changed this :)